test/unit/points/domain/test-tier.js

```diff
@@ -9,7 +9,7 @@ describe('points/domain/Tier', function () {
     expect(function () { new Tier(undefined, 1) }).to.throw(Error)
   })
   it('can retrieve points when they are defined', function () {
-    var tier = new Tier(new TierCount(4, 1, 1, 1), 4)
-    expect(tier.points).to.equal(4)
+    var tier = new Tier(new TierCount(4, 1, 1, 1))
+    expect(tier.tierCounts).to.be.an('object')
```

Comment:
Is this test still the same as its description? Maybe it needs renamed or updated to test it can retrieve the points. Can't fully remember to make a suggestion. Sorry.